sass/themes/hsl/_theme.scss

```diff
@@ -46,6 +46,7 @@ $metro-color: $hsl-metro-orange;
 $rail-color: $hsl-rail-red;
 $ferry-color: $hsl-ferry-blue;
 $wait-color: #aaa;
+$external-feed-color: #c5cad2;
 
```

Comment:
This seems to be the same value as in default theme. I wonder if it is necessary to repeat the value here -  maybe it gets inherited from  the default theme.

Reply:
The idea behind this was that because the changes were only to the HSL config, the value is supposed to be in the hsl theme. However, without a default value in the default css config, there is an error. I removed this from the HSL theme now